contracts/axelar-gateway/src/auth.rs

```diff
@@ -0,0 +1,211 @@
+use axelar_soroban_interfaces::types::{ProofSignature, ProofSigner, WeightedSigner};
+use soroban_sdk::xdr::ToXdr;
+use soroban_sdk::{crypto::Hash, panic_with_error, Bytes, BytesN, Env, Vec};
+
+use crate::storage_types::AuthDataKey;
+use crate::{error::AuthError, event, auth};
+use axelar_soroban_interfaces::types::{Proof, WeightedSigners};
+
+pub(crate) fn init_auth_verifier(
```

Comment:
```suggestion
pub fn init_auth_verifier(
```
`crate` scoping isn't needed on each function, since the whole module visibility is controlled via `mod auth` instead of `pub mod auth`

Reply:
updated.